tensorflow/python/layers/convolutional.py

```diff
@@ -1149,7 +1139,8 @@ def conv2d_transpose(inputs,
                      trainable=True,
                      name=None,
                      reuse=None):
-  """Transposed convolution layer (sometimes called Deconvolution).
+  """Functional interface for transposed 2D convolution layer (sometimes
+  called 2D Deconvolution).
```

Comment:
The one-line docstring summary should fit on one line. No need to mention "sometimes called 2D Deconvolution".